CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+0.6.0
+
+- Previously `drf-oidc-auth` was a hard dependency in `django-helusers`. That's no longer the case. Within `django-helusers` the `drf-oidc-auth` package is only used by the `helusers.oidc.ApiTokenAuthentication` class. If you want to keep on using that class, make sure you bring `drf-oidc-auth` into your project as an explicit dependency.
```

Comment:
You should also mention this in the README.

Reply:
Yep. The README should be improved a lot in other ways too.

Reply:
For this PR, it's enough just to document this detail, which is important for most users right now.

Reply:
Updated `README.md`.